plugins/modules/user.py

```diff
@@ -165,11 +165,6 @@
 }
 
 SPEC = {
```

Comment:
I'm seeing an error on the user_basic test now:

```
TASK [user_basic : Create Linode User] *****************************************
fatal: [testhost]: FAILED! => {"changed": false, "msg": "failed to create user: AccountGroup.user_create() got an unexpected keyword argument 'state'"}
```

I'm assuming the state just needs to be filtered from the creation kwargs 👍